apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.types.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+
+export const AssessmentAccessRulesSchema = z.object({
+  mode: z.string(),
+  uids: z.string(),
+  start_date: z.string(),
+  formatted_start_date: z.string().optional(),
+  end_date: z.string(),
+  formatted_end_date: z.string().optional(),
```

Comment:
Rather than passing around `formatted_start_date` and `formatted_end_date` everywhere, it'd be better to pass around just `start_date` and `end_date` and format them when needed.

Reply:
Updated to just format `start_date` and `end_date` as needed in [1dfef07](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/1dfef07527ed0296ff3cdfa7c58d85fce2304b97) and [0cad416](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/0cad41697059fbc8d4db96fc0dc903350be1488f).